adafruit_ble_radio.py

```diff
@@ -158,7 +166,9 @@ def receive(self, timeout=1):
             return msg[0].decode("utf-8").replace("\x00", "")
         return None
 
-    def receive_full(self, timeout=1):
+    def receive_full(
+        self, timeout: float = 1.0
+    ) -> Optional[tuple[ReadableBuffer, int, float]]:
```

Comment:
So this is an interesting one!  This is actually correct, since PEP 585 made it so builtins like `tuple` could be used as generics for typing instead of their `typing` module counterparts.

However, this was only added in Python 3.9, and Blinka currently supports Python 3.7 as a minimum.  So unfortunately, we'll have to using the `typing.Tuple` counterpart :'(

```suggestion
    ) -> Optional[Tuple[ReadableBuffer, int, float]]:
```

Don't forget to import `Tuple` from the typing module at the top!